samples/voting/README.md

```diff
@@ -9,11 +9,44 @@ The project should be immediately runnable by calling `tye run` from the directo
 
 A few things need to be configured before deploying to Kubernetes.
 
-- Setting up Redis. A connection string needs to be provided to connect to Redis. You can follow our tutorial on [setting up redis in your cluster](../../docs/tutorials/hello-tye/02_add_redis.md).
-- Setting up postgresql. A connection string eventually needs to be provided to tye for postgresql.
-- Deploying the ingress.yaml by calling:
+- [Create an Azure container registry using the Azure portal](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal) or you can use your public [docker hub account](https://hub.docker.com/)
+
+- `Redis` can be deployed using the below command :
 
     ```
-    kubectl apply -f ingress.yml
+    kubectl apply -f https://raw.githubusercontent.com/dotnet/tye/master/docs/tutorials/hello-tye/redis.yaml
+    ```
+- `Postgresql` can be installed by the following command :
+
+    ```
+    kubectl apply -f https://raw.githubusercontent.com/dotnet/tye/master/docs/tutorials/hello-tye/postgres.yaml
     ```
+
+- Once the deployment is done, you need to keep a note of the below connection strings :
+
+    -  `redis:6379`
+    -  `Server=postgres;Port=5432;User Id=postgres;Password=pass@word1;`
+
+>! NOTE: If you want, you can modify the password in `postgres` yaml.
+
+- You need to install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/) in your Kubernetes cluster. 
+  You can also use a package manger like `helm` to [create an ingress controller](https://docs.microsoft.com/en-us/azure/aks/ingress-basic#create-an-ingress-controller) to expose your application outside of your Kubernetes cluster. 
+  
+  And then you need to expose those services by using ingress yaml. 
     
+    ```
+    kubectl apply -f ingress.yml
+    ```
+
+- After that, run `tye deploy --interactive --namespace default` to do all the deployment.
```

Comment:
Why --namespace default?

Reply:
Well this strange while using `tye deploy --interactive` I am getting below error while deploying to both local k8s and AKS. 

![image](https://user-images.githubusercontent.com/13281246/98630513-558bbe00-2341-11eb-8177-1b53c731399e.png)

But when I provide the namespace value everything seems to be fine. That's the reason I had included that `--namespace` parameter.

Reply:
Ah, did you set your current namespace in your kube context to teractive?

Reply:
No, actually I found out that kubectl current context wasn't even set to any namespace. That's the reason it was throwing that error. I am not sure from where the name `teractive` was coming up in that error message. But when I set my current kubectl context to namespace `default` then I was able to deploy the sample with `tye deploy --interactive`. 

I have updated the `Readme` file as well. If you could review that again.

Reply:
After resetting my local kubernetes cluster I was able to deploy it to the local K8s without any issue with help of `tye deploy -i` command.  So it seems to be the problem of my local environment. Updated the readme file.